db/migrate/20210318194322_change_data_type_for_price.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDataTypeForPrice < ActiveRecord::Migration[6.0]
+  def change
+    change_column :bars, :price, :string
```

Comment:
why is there a change? Price should be :decimal type not string  (Cataline told us that)